plugin/package.json

```diff
@@ -10,8 +10,9 @@
     "prod": "webpack --config webpack.prod.js",
     "start": "yarn dev --watch",
     "lint:js": "eslint src/main/js --ext js",
+    "lint:js-ci": "eslint src/main/js -f checkstyle -o target/eslint-warnings.xml --ext js",
     "mvnbuild": "yarn prod",
-    "mvntest": "yarn lint:js"
+    "mvntest": "npx if-env CI=true && yarn lint:js-ci || yarn lint:js"
```

Comment:
Does this work on Windows, or is this part of the build even run on Windows?

Reply:
didn't work on windows:

```
[2024-09-23T15:43:25.323Z] [INFO] $ npx if-env CI=true && yarn lint:js-ci || yarn lint:js
[2024-09-23T15:43:25.323Z] [INFO] 'npx' is not recognized as an internal or external command,
[2024-09-23T15:43:25.323Z] [INFO] operable program or batch file.
```

Reply:
And yet the build passes?

I think it is fine to run validation tools like these only on Linux (they are not likely to be platform-dependent), just surprised that this does not cause a build failure.

Reply:
yes, the next parent pom version by default would cause it to fail with this PR: https://github.com/jenkinsci/plugin-pom/pull/1008

Reply:
It worked without `npx`